src/ethereum/api/graphClient.ts

```diff
@@ -1,9 +1,13 @@
+import nodefetch, { RequestInit } from 'node-fetch'
```

Comment:
Would you mind checking out if there's any other place where isomorphic fetch is used so we can remove it now? Thanks to this change, we no longer need the global fetch.
We can also try removing the `dom` keyword from the libraries section.